environments.json

```diff
@@ -1458,10 +1458,25 @@
   "node7": {
     "full": "Node.js",
     "family": "Node.js",
-    "short": "Node 7",
+    "short": "Node 7.0-7.5",
```

Comment:
I see. What exactly is needed for one environment to extend results of another by default?

This shouldn't be specific to my PR, the Node 6/6.5 setup should share those problems.